src/Microsoft.VisualStudio.SDK.Analyzers.CodeFixes/VSSDK001DeriveFromAsyncPackageCodeFix.cs

```diff
@@ -93,10 +91,10 @@ private async Task<Document> ConvertToAsyncPackageAsync(CodeFixContext context,
             };
             nodesToTrack.AddRange(getServiceInvocationsSyntax);
             nodesToTrack.RemoveAll(n => n == null);
-            SyntaxNode updatedRoot = root.TrackNodes(nodesToTrack);
+            SyntaxNode updatedRoot = root.TrackNodes(nodesToTrack!);
 
             // Replace the Package base type with AsyncPackage
-            baseTypeSyntax = updatedRoot.GetCurrentNode(baseTypeSyntax);
+            baseTypeSyntax = updatedRoot.GetCurrentNode(baseTypeSyntax)!;
```

Comment:
We could perhaps swap out the `!` for an added `Assumes.NotNull(baseTypeSyntax)` statement after this one so we get a clearer exception thrown instead of NRE if this assumption ever fails.

Same below as well.

Reply:
Done